### ISSUE TITLE
Doesn't detect HP Deskjet 2130 scanner

### ISSUE THREAD

**Issue by bbhtt** [flathub/org.gnome.SimpleScan#9]
Hello, the flatpak package of Simple scan doesn't seem to detect my HP Deskjet 2130 printer and scanner. If it is installed from the distribution repository it works fine.

Here are the logs with `--debug`:

Flatpak:

```
➜ flatpak run org.gnome.SimpleScan --debug
[+0.00s] DEBUG: simple-scan.vala:2015: Starting simple-scan 44.0, PID=2
[+0.00s] DEBUG: unsetenv() is not thread-safe and should not be used after threads are created
[+0.12s] DEBUG: _g_io_module_get_default: Found default implementation keyfile (GKeyfileSettingsBackend) for ‘gsettings-backend’
[+0.14s] DEBUG: app-window.vala:2002: Loading state from /home/wirt/.var/app/org.gnome.SimpleScan/config/simple-scan/state
[+0.14s] DEBUG: app-window.vala:1981: Restoring window to 600x410 pixels
[+0.15s] DEBUG: scanner.vala:1619: sane_init () -> SANE_STATUS_GOOD
[+0.15s] DEBUG: scanner.vala:1625: SANE version 1.2.1
[+0.15s] DEBUG: scanner.vala:1686: Requesting redetection of scan devices
[+0.15s] DEBUG: scanner.vala:863: Processing request
[+0.30s] DEBUG: app-window.vala:2078: Saving state to /home/wirt/.var/app/org.gnome.SimpleScan/config/simple-scan/state
[+0.46s] DEBUG: app-window.vala:2078: Saving state to /home/wirt/.var/app/org.gnome.SimpleScan/config/simple-scan/state
[+3.01s] DEBUG: scanner.vala:348: sane_get_devices () -> SANE_STATUS_GOOD
[+3.01s] DEBUG: scanner.vala:1686: Requesting redetection of scan devices
[+3.01s] DEBUG: scanner.vala:863: Processing request

...
```

Distribution:

```
➜ simple-scan --debug
[+0.00s] DEBUG: simple-scan.vala:2015: Starting simple-scan 44.0, PID=6451
[+0.00s] DEBUG: unsetenv() is not thread-safe and should not be used after threads are created
[+0.14s] DEBUG: app-window.vala:2002: Loading state from /home/wirt/.config/simple-scan/state
[+0.14s] DEBUG: app-window.vala:1981: Restoring window to 600x410 pixels
[+0.15s] DEBUG: scanner.vala:1619: sane_init () -> SANE_STATUS_GOOD
[+0.15s] DEBUG: scanner.vala:1625: SANE version 1.2.1
[+0.15s] DEBUG: scanner.vala:1686: Requesting redetection of scan devices
[+0.15s] DEBUG: scanner.vala:863: Processing request
[+0.30s] DEBUG: app-window.vala:2078: Saving state to /home/wirt/.config/simple-scan/state
[+0.46s] DEBUG: app-window.vala:2078: Saving state to /home/wirt/.config/simple-scan/state
[+4.03s] DEBUG: scanner.vala:348: sane_get_devices () -> SANE_STATUS_GOOD
[+4.03s] DEBUG: scanner.vala:374: Device: name="hpaio:/usb/DeskJet_2130_series?serial=CN654480H9065Z" vendor="Hewlett-Packard" model="DeskJet_2130_series" type="all-in-one"
[+4.36s] DEBUG: app-window.vala:2078: Saving state to /home/wirt/.config/simple-scan/state
[+6.90s] DEBUG: autosave-manager.vala:201: Deleting autosave records
[+6.90s] WARNING: autosave-manager.vala:214: Failed to delete autosaves: Error opening directory “/home/wirt/.cache/simple-scan/autosaves”: No such file or directory
[+6.90s] DEBUG: scanner.vala:1807: Stopping scan thread
[+6.90s] DEBUG: scanner.vala:863: Processing request
[+6.91s] DEBUG: scanner.vala:1818: sane_exit ()
```

The printer is connected over USB, it doesn't support network connection.

Host: 

```
➜ lsusb|grep HP      
   
Bus 003 Device 002: ID 03f0:e111 HP, Inc DeskJet 2130 series

➜ sane-find-scanner|grep HP

found possible USB scanner (vendor=0x03f0 [HP], product=0xe111 [DeskJet 2130 series]) at libusb:003:002
```

Flatpak:

```
[📦 org.gnome.SimpleScan ~]$ sane-find-scanner|grep HP

found possible USB scanner (vendor=0x03f0 [HP], product=0xe111 [DeskJet 2130 series]) at libusb:003:002
```

Thanks!

**Comment by hfiguiere** [comment]
as mentionned in the original submission review, building more of the sane backends would make it more usefull.

**Comment by bbhtt** [comment]
No dice with https://github.com/flathub/org.gnome.SimpleScan/pull/19

**Comment by hfiguiere** [comment]
See also #22

**Comment by eheintzmann** [comment]
See #23

**Comment by hfiguiere** [comment]
Fixed by #23
